### PR TITLE
[DE] better understandability of 'doing' and removing mandatory article for area

### PR DESCRIPTION
**Pull request by NebuXIII**
adding for better acoustic intelligibility of 'doing'(german: 'machen'; since whisper always understands 'macht') and correcting mandatory article for areas

### CODE REVIEW

File: sentences/de/_common.yaml [hunk 1]
```diff
@@ -172,7 +172,7 @@ expansion_rules:
   schliessen: "(schlie(ß|ss)[e|en]|zumachen|zu machen)"
   öffnen: "(öffne[n]|aufmachen|auf machen)"
   schalten: "(schalt[e|en])"
-  machen: "(mach[e|en])"
+  machen: "(mach[e|en|t])"
```
Comment (mib1185) [inline sentences/de/_common.yaml:175]: we need new test sentences for this change

File: sentences/de/_common.yaml [hunk 2]
```diff
@@ -162,7 +162,7 @@ expansion_rules:
   artikel_unbestimmt: "[(ein|eine|eines|einer|einem|einen)]"
   artikel: "[<artikel_bestimmt>|<artikel_unbestimmt>]"
   name: "<artikel> {name}"
-  area: "([(in|an|auf)] <artikel>|im|am|<von_dem>) {area}[s]"
+  area: "[([(in|an|auf)] <artikel>|im|am|<von_dem>)] {area}[s]"
```
Comment (mib1185) [inline sentences/de/_common.yaml:165]: please move this change to a separate PR and explain the intention or goal of this change. further we also need test sentences for this change

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @NebuXIII 
thanks for your contribution.
Please have a look at the comments.
Thx 👍

File: sentences/de/_common.yaml (hunk 2 above)
Comment (steffenrapp) [reply sentences/de/_common.yaml:165]: That should not be mandatory right now. There are many brackets which might be confusing.
I have also changed something here in #1657

**Comment by NebuXIII** [comment]
Hi @mib1185 ,
thank you for your feedback, I will do so!

File: sentences/de/_common.yaml (hunk 1 above)
Comment (mib1185) [reply sentences/de/_common.yaml:175]: this part is meanwhile superseded by #1725

**Comment by frenck** [comment]
Because there hasn't been any activity on this PR for quite some time now, I've decided to close it for being stale.

Feel free to re-open this PR when you are ready to pick up work on it again 👍

../Frenck
